From bb84d3ef336f85d8f796fbe5a600791894c82a47 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 24 Jul 2018 11:17:01 +0800
Subject: [PATCH 205/205] ARM: OMAP2+: sleep33xx: set emif_sram_table align
 with 8 bits

Since sram-exec uses aligned fncpy instead of memcpy with 8 bits, so we
have to set the same align for emif_sram_table to avoid emif_sram crash
when loading suspend codes.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/sleep33xx.S |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-omap2/sleep33xx.S b/arch/arm/mach-omap2/sleep33xx.S
index 854363b..a56a972 100644
--- a/arch/arm/mach-omap2/sleep33xx.S
+++ b/arch/arm/mach-omap2/sleep33xx.S
@@ -250,6 +250,7 @@ module_disabled_val:
 	.word	0x30000
 
 /* DDR related defines */
+	.align 3
 am33xx_emif_sram_table:
 	.space EMIF_PM_FUNCTIONS_SIZE
 
-- 
1.7.9.5

